index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Messi Fever!</title>
+
+		<link rel="stylesheet" type="text/css" href="style.css">
+ 	</head>
+	
+	<body>
+		
+		<nav>
+			<a href="index.html"> Home </a> |
+			<a href="experience/experience.html" target="_blank"> Resume and Honors</a> |
+			<a href="http://www.fcbarcelona.com/all-about-leo-messi" target="_blank"> Official Messi site</a> |
+			<a href="http://www.worldsoccershop.com/shop-by-player-messi--lionel.html" target="_blank"> Buy Gear </a>
+		</nav>
+
+		<h1>Messi Fever! </h1>
+		<img src="lionel messi cover.jpeg" text="Lionel Messi" class="main" >
+		<h3>La Pulga "The Flea"</h3>
+		
+
+		<hr/>
+		<h3 class="flea">
+			What's that? The Flea? 
+			</h3>
+		
+ 		<p>
```

Comment:
:hammer: These empty lines within the paragraph don't actually translate onto the web page. If you want it to look like that, you'd have make new `<p>`'s for each line. Otherwise, best practice would remove these unnecessary empty lines.